pip/pip-342 OTel client metrics support.md

```diff
@@ -0,0 +1,167 @@
+| `pulsar.client.consumer.message.ack`            | Counter       | messages    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of acknowledged messages                                                                                                       |
+| `pulsar.client.consumer.message.nack`           | Counter       | messages    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of negatively acknowledged messages                                                                                            |
+| `pulsar.client.consumer.message.dlq`            | Counter       | messages    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of messages sent to DLQ                                                                                                        |
+| `pulsar.client.consumer.message.ack.timeout`    | Counter       | messages    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of messages that were not acknowledged in the configured timeout period, hence, were requested by the client to be redelivered |
+| `pulsar.client.producer.message.send.duration`  | Histogram     | seconds     | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`]                                                                            | Publish latency experienced by the application, includes client batching time                                                             |
+| `pulsar.client.producer.rpc.send.duration`      | Histogram     | seconds     | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.response.status="success\|failed"`                                | Publish RPC latency experienced internally by the client when sending data to receiving an ack                                            |
+| `pulsar.client.producer.message.send.size`      | Counter       | bytes       | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.response.status="success\|failed"`                                | The number of bytes published                                                                                                             |
```

Comment:
Is there a reason why we don't have producer message send count?

Reply:
The send latency histogram includes a "count", which can be used to compute the publish rate